pvr.nextpvr/addon.xml.in

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.nextpvr"
-  version="21.1.1"
+  version="21.1.2"
```

Comment:
If you add new features to the add-on, minor version should be increased. Micro bump is for bug fix only releases.

Reply:
Definitely.  I had bumped it but then it got confused after the bump for translations conflict.